### PR TITLE
Update Proton Mail icon

### LINKED ISSUE #8649: Proton Mail is outdated
### Brand Name

Proton mail

### Official Resources for Icon and Color

https://proton.me/media/kit

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
protonmail

**Issue:** closes #8649 

**Similarweb rank:** 137,199. See https://www.similarweb.com/website/protonmail.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from Official Icons in Brand website https://proton.me/media/kit
Color: `#9d82ff` Taken from averaging logo colors by square root method.

### CODE REVIEW

**Comment by LitoMore** [comment]
Duplicate of #7793

**Comment by LitoMore** [comment]
We can still review and merge this PR. And update the title in another PR and mark the title change as a breaking change.

**Comment by LitoMore** [comment]
Let's try with a 2px gap. 2px will be a better look.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9356,8 +9356,8 @@
         },
         {
             "title": "ProtonMail",
-            "hex": "8B89CC",
-            "source": "https://protonmail.com/media-kit"
+            "hex": "9D82FF",
```
Comment (LitoMore) [inline _data/simple-icons.json:9359]: I'd prefer to use their theme color #6d4aff due to the gradient being hard to choose.

```suggestion
            "hex": "6D4AFF",
```

**Comment by PratyushJayachandran** [comment]
> Let's try with a 2px gap. 2px will be a better look.

2px seemed to be a bit much. I did a 1px gap. It looks good to me. What is your opinion?
protonmail

**Comment by PratyushJayachandran** [comment]
> We can still review and merge this PR. And update the title in another PR then mark the title change as a breaking change.

I didn't understand this part.

By the way I created this PR as I saw that old PR was stuck at requested change for too long.

**Comment by LitoMore** [comment]
See https://github.com/simple-icons/simple-icons/blob/develop/VERSIONING.md.

**Comment by PratyushJayachandran** [comment]
> See https://github.com/simple-icons/simple-icons/blob/develop/VERSIONING.md.

Oh now I get it. You were taking about the title in JSON. So we will be putting a space between Proton and Mail?
If we are doing that, that will be part of a Major release, correct?

**Review by LitoMore — APPROVED** [review]
image

Looks good.

**Comment by LitoMore** [comment]
@PratyushJayachandran We could update the Proton Drive to 2px gap as well.

And the Proton Calendar needs to be added to our library. PRs welcome.

image

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran We could update the Proton Drive to 2px gap as well.
> 
> And the Proton Calendar needs to be added to our library. PRs welcome.
> image

Proton Calendar PR: https://github.com/simple-icons/simple-icons/pull/8883

Proton Drive: Soon™️

**Pull request merged**